buffer_test.ts

```diff
@@ -17,7 +17,7 @@ function init() {
   if (testBytes == null) {
     testBytes = new Uint8Array(N);
     for (let i = 0; i < N; i++) {
-      testBytes[i] = "a".charCodeAt(0) + i % 26;
+      testBytes[i] = "a".charCodeAt(0) + (i % 26);
```

Comment:
I think prettier might have erroneously removed that when I landed 2696658
Have your run prettier?

Reply:
Yeah this was changed by my local prettier. Probably I used a different config?

Reply:
I can confirm prettier removes it